influence_benchmark/utils/wandb_logging.py

```diff
@@ -77,14 +80,21 @@ def format_stats_html(stats):
     return stats_html
 
 
-def extract_wandb_data(df):
+def extract_wandb_data(df, final_reward):
     trajectories = []
 
+    if final_reward:
+        reward_label = "traj_final_rew"
+        influence_label = "traj_final_infl"
+    else:
+        reward_label = "traj_mean_rew"
+        influence_label = "traj_mean_infl"
```

Comment:
I'm not sure if we should have different labels. I'm tempted to just call this "cumulative reward" or "utility" and call it a day, implying that this may be measured in different ways